<p>##Factors that influence linkage disequilibrium
<a href="http://www.nature.com/nrg/journal/v3/n4/box/nrg777_BX1.html"><strong>Source</strong></a></p>

<p>Mutation and recombination might have the most evident impact on linkage disequilibrium (LD), but there are additional contributors to the extent and distribution of disequilibrium. Most of these involve demographic aspects of a population, and tend to sever the relationship between LD strength and the physical distance between loci.</p>

<ol>
  <li>
    <p><strong>Genetic drift</strong>. This phenomenon describes the change in gene and haplotype frequency in a population every generation owing to the random sampling of gametes that occurs during the production of a finite number of offspring. Frequency changes are accentuated in small populations. In general, the increased drift of small, stable (not growing) populations tends to increase LD, as haplotypes are lost from the population. Such populations might be suitable for disease-gene mapping, with the idea that genetic drift will accentuate disease and marker allele frequency differences between cases and controls. However, the applicability of this phenomenon to gene mapping has not been well characterized.</p>
  </li>
  <li>
    <p><strong>Population growth</strong>. Rapid population growth decreases LD by reducing genetic drift.</p>
  </li>
  <li>
    <p><strong>Admixture or migration</strong>. LD can be created by ADMIXTURE, or by migration (gene flow), between populations. Initially, LD is proportional to the allele frequency differences between the populations, and is unrelated to the distance between markers. In subsequent generations, the ‘spurious’ LD between unlinked markers quickly dissipates, while LD between nearby markers is more slowly broken down by recombination. In theory, this would allow the mapping of disease genes in hybrid populations without using many genetic markers. Several admixed populations, such as African Americans and Hispanic Americans, have been characterized with this application in mind, but the success of this approach will depend heavily on the time since admixture occurred, the frequency differences of the disease of interest in the parental populations and the allele frequency differences. So, the diseases and circumstances for which this mapping approach will be feasible might turn out to be quite rare and exceptional.</p>
  </li>
  <li>
    <p><strong>Population structure</strong>. Various aspects of population structure are thought to influence LD. Population subdivision is likely to have been an important factor in establishing the patterns of LD in humans, but most of our limited knowledge comes from the study of model organisms. An interesting recent study of Arabidopsis indicated that extreme inbreeding can produce high levels of LD without a substantial reduction in levels of variation. This neglected area would benefit from intensified study in humans.</p>
  </li>
  <li>
    <p><strong>Natural selection</strong>. There are two primary routes by which selection can affect the extent of disequilibrium. The first is a hitchhiking effect, in which an entire haplotype that flanks a favoured variant can be rapidly swept to high frequency or even fixation. Although the effect is generally milder, selection against deleterious variants can also inflate LD, as the deleterious haplotypes are swept from the population. The second way in which selection can affect LD is through epistatic selection for combinations of alleles at two or more loci on the same chromosome. This form of selection leads to the association of particular alleles at different loci. Although this has provided a major motivation for historical studies of LD in Drosophila genetics, as a means of detecting the action of (epistatic) natural selection, it has not yet been shown to alter LD in humans.</p>
  </li>
  <li>
    <p><strong>Variable recombination rates</strong>. Recombination rates are known to vary by more than an order of magnitude across the genome. Because breakdown of LD is primarily driven by recombination, the extent of LD is expected to vary in inverse relation to the local recombination rate. It is even possible that recombination is largely confined to highly localized recombination hot spots, with little recombination elsewhere. According to this view, LD will be strong across the non-recombining regions and break down at hot spots. Although there are intriguing indications that this reflects the situation for some regions, the generality of the hot-spot phenomenon, the strength of recombination in and outside hot spots, and the length distributions of these regions remain to be determined.</p>
  </li>
  <li>
    <p><strong>Variable mutation rates</strong>. Some single-nucleotide polymorphisms, such as those at CpG dinucleotides, might have high mutation rates and therefore show little or no LD with nearby markers, even in the absence of historical recombination.</p>
  </li>
  <li>
    <p><strong>Gene conversion</strong>. In a gene conversion event, a short stretch of one copy of a chromosome is transferred to the other copy during meiosis. The effect is equivalent to two very closely spaced recombination events, and can break down LD in a manner similar to recombination or recurrent mutation. It has recently been shown that rates of gene conversion in humans are high and are important in LD between very tightly linked markers.</p>
  </li>
</ol>

<p><em>Quoted from Kristin G. Ardlie, Leonid Kruglyak &amp; Mark. 2002. Patterns of linkage disequilibrium in the human genomeSeielstad. Nature Reviews Genetics 3, 299-309 doi:10.1038/nrg777</em></p>
